pkg/commands/standard/raw_test.go

```diff
@@ -0,0 +1,47 @@
+package standard
+
+import (
+	"github.com/stretchr/testify/assert"
+	"io/ioutil"
+	"path/filepath"
+	"runtime"
+	"strconv"
+	"testing"
+)
+
+func TestGetJsonBodyFromFlags(t *testing.T) {
+	for i, testcase := range []struct {
+		fileName         string
+		bodyString       string
+		params           map[string]string
+		expectedBody     string
+		expectedMimeType string
+	}{
+		{"raw_test_body.json", "", nil, "{\"test\": \"yes it is\"}\n", "application/json"},
+		{"raw_test_body.qvf", "", nil, "<qvf content placeholder>\n", "binary/octet-stream"},
+		{"raw_test_body-image.png", "", nil, "<skip assert>", "image/png"},
+		{"", "inline-body-string", nil, "inline-body-string", "text/plain"},
+		{"", "{\"test\": \"yes it is\"}", nil, "{\"test\": \"yes it is\"}", "application/json"},
+		{"", "", map[string]string{"key": "value"}, "{\"key\":\"value\"}", "application/json"},
+		{"", "", map[string]string{"key:bool": "true"}, "{\"key\":true}", "application/json"},
+		{"", "", map[string]string{"nested.key": "value"}, "{\"nested\":{\"key\":\"value\"}}", "application/json"},
```

Comment:
Since we'll be parsing user input here, maybe we want some negative test cases as well?
`key:bla` `nested.key.`